packages/app-desktop/app.ts

```diff
@@ -73,7 +73,6 @@ import SearchEngine from '@joplin/lib/services/search/SearchEngine';
 import { PackageInfo } from '@joplin/lib/versionInfo';
 import { CustomProtocolHandler } from './utils/customProtocols/handleCustomProtocols';
 import { refreshFolders } from '@joplin/lib/folders-screen-utils';
-
 const pluginClasses = [
```

Comment:
**🔵 Optional/style**: This whitespace change seems unrelated to the pull request. Consider reverting it.

Reply:
Fixed in commit 6087f771b63139d7950cae5e8abaa4a0a7a0ca7f.